main/data/default_encryption_dialog.ui

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.2 -->
+<interface>
+  <requires lib="gtk+" version="3.6"/>
+  <object class="GtkDialog" id="dialog">
+    <property name="can_focus">True</property>
+    <property name="modal">True</property>
+    <property name="default_width">320</property>
+    <property name="default_height">260</property>
+    <child type="titlebar">
+      <placeholder/>
+    </child>
+    <child>
+      <object class="GtkBox">
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkBox">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="orientation">vertical</property>
+            <child>
+              <object class="GtkLabel" id="default_encryption_warning_label">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="label" translatable="yes">No default end-to-end encryption method has been previously selected for this conversation. XMPP defines some XEP for end-to-end encryption that Dino supports.
+
+It is strongly recommended that one of the following end-to-end encryption methods below is selected before sending a message.
+
+TAKE INTO ACCOUNT UNENCRYPTED CONVERSATIONS COULD BE READ BY THIRD PARTIES.
+
+These settings can be later changed from the Settings menu.</property>
+                <property name="wrap">True</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkCheckButton" id="omemo">
+                <property name="label" translatable="yes">OMEMO (XEP-0384)</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">False</property>
+                <property name="active">False</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkCheckButton" id="openpgp">
+                <property name="label" translatable="yes">OpenPGP (XEP-0027)</property>
```

Comment:
See my other comments for the reasoning behind the suggestion
```suggestion
                <property name="label" translatable="yes">OpenPGP (manual setup)</property>
```

Alternatively this, but it might be too long:
```suggestion
                <property name="label" translatable="yes">OpenPGP (external setup required)</property>
```

Reply:
As opposed to `manual setup`, I think `external setup required`, while longer, makes clear the fact external tools must be used to set up OpenPGP encryption.